src/components/AppLayout/AppHeader.tsx

```diff
@@ -285,7 +305,7 @@ export function AppHeader({
         href: '/generate',
         visible: !isMuted,
         label: (
-          <Group align="center" spacing="xs">
+          <Group className="flex flex-col md:flex-row items-center gap-2 text-xs md:text-md">
```

Comment:
Seems like we can remove the Group component and reuse the className for all other items here
```suggestion
          // Somewhere outside the component
          const menuItemClassName = 'flex flex-col md:flex-row items-center gap-2 text-xs md:text-md';
          
          <div className={menuItemClassName}>
```
Or create a simple component to reuse here, just want to avoid repetitiveness 😬